README.md

```diff
@@ -1,219 +1,24 @@
 Distributed System Integration & Performance Testing Library
 ============================================================
 
-This repository contains tools for running system integraton and performance
-tests. It provides utilities for pulling up and tearing down services
-easily, using Vagrant to let you test things on local VMs or run on EC2
-nodes. Tests are just Python scripts that run a set of services, possibly
-triggering special events (e.g. bouncing a service), collect results (such as
-logs or console output) and report results (expected conditions met, performance
-results, etc.).
+Overview
+--------
 
+Ducktape contains tools for running system integration and performance tests. It provides the following features:
 
-Install
--------
-
-Note: as a general rule, it's recommended to use an isolation tool such as `virtualenv`.
-
-Prerequisites:
-* ducktape uses paramiko, which depends upon cryptography, which has non-python external requirements
-* https://cryptography.io/en/latest/installation/
-* OSX should just work
-* Ubuntu:
-
-    ```
-    $ sudo apt-get install build-essential libssl-dev libffi-dev python-dev
-    ```
-* Fedora and RHEL-derivatives:
-
-    ```
-    $ sudo yum install gcc libffi-devel python-devel openssl-devel
-    ```
-
-
-Install ducktape: `pip install ducktape`
-
-On macOS you may need to `brew install openssl` and then install using `C_INCLUDE_PATH=/usr/local/opt/openssl/include
-LIBRARY_PATH=/usr/local/opt/openssl/lib pip install ducktape`.
-
-Use
----
-ducktape discovers and runs tests in the path provided.
-
-    ducktape <relative_path_to_testdirectory>               # e.g. ducktape dir/tests
-    ducktape <relative_path_to_file>                        # e.g. ducktape dir/tests/my_test.py
-    ducktape <path_to_test>[::SomeTestClass]                # e.g. ducktape dir/tests/my_test.py::TestA
-    ducktape <path_to_test>[::SomeTestClass[.test_method]]  # e.g. ducktape dir/tests/my_test.py::TestA.test_a
-
-Use the `--collect-only` flag to discover tests without running any:
-
-    ducktape <path_to_testfile_or_directory> --collect-only
-
-Use the `--parameters` flag followed by a JSON string to specify which parameters to use when running the discovered test:
+* Isolation by default so system tests are as reliable as possible.
+* Utilities for pulling up and tearing down services easily in clusters in different environments (e.g. local,
+* custom cluster, Vagrant, K8s, Mesos, Docker, cloud providers, etc.)
+* Easy to write unit tests for distributed systems
```

Comment:
I'd probably phrase this a bit differently since this is kind of confusing (a unit test by definition can't really be for a distributed system). Perhaps something more like:

```
* Write tests for distributed systems in a simple unit test-like style
```